tsdb/points.go

```diff
@@ -625,6 +635,13 @@ func escapeString(in string) string {
 	return in
 }
 
+func escapeStringFields(in string) string {
```

Comment:
Looks like a good case for [strings.Replacer](http://golang.org/pkg/strings/#Replacer)